engine/src/main/java/com/redhat/cloud/notifications/models/EmailAggregationKey.java

```diff
@@ -43,18 +50,19 @@ public boolean equals(Object o) {
         }
 
         EmailAggregationKey that = (EmailAggregationKey) o;
-        return Objects.equals(accountId, that.accountId) && Objects.equals(bundle, that.bundle) && Objects.equals(application, that.application);
+        return Objects.equals(accountId, that.accountId) && Objects.equals(bundle, that.bundle) && Objects.equals(application, that.application) && Objects.equals(orgId, that.orgId);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(accountId, bundle, application);
+        return Objects.hash(accountId, orgId, bundle, application);
```

Comment:
@josejulio should we remove it from here for now as well?

Reply:
I guess it would make sense to remove it from here until we start using it.